Automattic/wp-calypso
# Logging: Show placeholder UI while loading logs

This PR should be merged after https://github.com/Automattic/wp-calypso/pull/75006, which it builds on.

Closes https://github.com/Automattic/dotcom-forge/issues/1995

## Proposed Changes

* Show a placeholder skeleton view of the table fetching data. Just basic bars for now. Maybe we can use columns in a later version. 


https://user-images.githubusercontent.com/6851384/228236254-bc086c76-d26e-496b-8e04-39d7a542b70c.mp4



## Testing Instructions



* Using this branch, navigate to `/site-logs/:my-atomic-site`
* Verify loading bars are shown like the video
* Try changing tabs or changing page and verify they are shown also. 

## Pre-merge Checklist
